entrypoint-legacy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+cat >&2 <<-EOF
++------------------------------------------------------------------------------+
+| WARNING: loadimpact/k6 Docker image has been replaced by grafana/k6.         |
+|          THIS IMAGE IS DEPRECATED and its support will be discontinued after |
+|          Dec 31, 2023. Please update your scripts to use grafana/k6 to       |
```

Comment:
That's a good timeframe :+1:

The idea is that by "discontinued" we mean that we will stop pushing new k6 releases to it, but it will still work with this warning. And some time after that (6 months?) we would push the `exit 1` change, and a bit more (6 more months?) we would delete the `loadimpact` namespace altogether.

Is this mostly the plan, pending timeframe agreements?

Reply:
I believe we can decide on how much do we want to push users to upgrade later. Personally I become more and more reluctant to adding `exit 1`. Not publishing new releases to the `loadimpact` org might be good enough.

Reply:
> Personally I become more and more reluctant to adding `exit 1`. Not publishing new releases to the loadimpact org might be good enough.

`exit 1` is part of making a gradual change. The first step is this deprecation warning; the next step is k6 tests failing with `exit 1` so that users will actually notice the warning; and the final step is removal of the `loadimpact` org, where CI builds would fail much earlier.

I suppose there's no harm in letting old images of k6 there forever, but we do want users to upgrade to new versions, and since Load Impact is not an entity anymore, there's no sense in keeping the org around. This way we avoid some Docker Inc. manager deciding again that maybe OSS projects should start paying for the data and traffic :sweat_smile:

Reply:
I believe once the EOL date has passed, we can retire `loadimpact` anytime should such decision be made.